### PR TITLE
Remove remaining traces of go2idl

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: staging/src/k8s.io/kube-gen/cmd/conversion-gen/generators/conversion.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/golang/glog"
 )
 
-// CustomArgs is used tby the go2idl framework to pass args specific to this
+// CustomArgs is used ty the gengo framework to pass args specific to this
```
Comment (ash2k) [inline staging/src/k8s.io/kube-gen/cmd/conversion-gen/generators/conversion.go]: oops

Comment (sttts) [inline staging/src/k8s.io/kube-gen/cmd/conversion-gen/generators/conversion.go]: :D

Comment (sttts) [inline staging/src/k8s.io/kube-gen/cmd/conversion-gen/generators/conversion.go]: Fixed

**Pull request merged**